core/vm/contracts_suave_eth.go

```diff
@@ -34,6 +38,7 @@ var (
 	extractHintAddress              = common.HexToAddress("0x42100037")
 	buildEthBlockAddress            = common.HexToAddress("0x42100001")
 	submitEthBlockBidToRelayAddress = common.HexToAddress("0x42100002")
+	submitEthBundleToBuilderAddress = common.HexToAddress("0x42100004")
```

Comment:
@ferranbt it'd be really nice if we could assign the addresses from suave/artifacts/addresses.go!